samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -250,12 +250,12 @@ The following table presents the patterns that you can use to switch antennas on
 | RFU    | 15 (0b1111)  |
 +--------+--------------+
 
-nRF21540 front-end module
-=========================
+Front-end module
+================
 
 .. include:: /includes/sample_dtm_radio_test_fem.txt
 
-You can configure the transmitted power gain, antenna output and activation delay in nRF21540 using vendor-specific commands, see `Vendor-specific packet payload`_.
+You can configure the transmitted power gain, antenna output, and activation delay in FEMs using vendor-specific commands, see `Vendor-specific packet payload`_.
```

Comment:
Please add a note to make it clear to the users, that each FEM has different interface and not all commands will work the same way, and some may not be supported, maybe something like:
Note: each front-end module (FEM) has different capabilities and operating modes, so some commands may not be supported by a specific FEM and those supported may work differently on different FEMs.

Reply:
Added